ete3/clustering/clustvalidation.py

```diff
@@ -192,13 +193,4 @@ def square_euclidean_dist(v1,v2):
         raise ValueError("Cannot calculate values")
     return  distance/valids
 
-try:
-   from scipy import stats
-except ImportError:
-    try:
-        from . import stats
-        default_dist = spearman_dist
-    except ImportError:
-        default_dist = euclidean_dist
-else:
-    default_dist = spearman_dist
+default_dist = spearman_dist
```

Comment:
I think nobody is no longer using this clustering-validation module. Having scipy as a dep just because we need spearman correlation is probably way too much. 
I agree on removing stats.py, but I would only raise an import exception if someone really tries to use the clustering validation functions and scipy is not present.